docker/setup-hosts.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+NODES=3
+
+
+## Create multiple hosts
+for N in $(seq 1 $NODES)
+do
+docker-machine create --driver virtualbox node$N
+done
+
+
+## Set up swarm
+### node1 as manager and others as workers
+SWARM_ADDR=$(docker-machine ip node1)
+
+docker-machine ssh node1 docker swarm init --advertise-addr $SWARM_ADDR
+
+SWARM_TOKEN=$(docker-machine ssh node1 docker swarm join-token -q worker)
+
+for N in $(seq 2 $NODES)
+do
+docker-machine ssh node$N docker swarm join --token $SWARM_TOKEN $SWARM_ADDR:2377
+done
+
+
+## Set up hosts
+for N in $(seq 1 $NODES)
+do
+docker-machine ssh node$N 'git clone https://github.com/lsst-dm/alert_stream.git && cd alert_stream \
```

Comment:
I avoided that at the moment because I'm not sure what the hub etiquette is here since ideally if using the hub it would be an automated build from github (so only one place to update), and I've not added an automated build here since it doesn't look like we are doing that yet (but there is one on my personal fork).